src/components/ModifyProfile/ModifyProfileInput.tsx

```diff
@@ -17,7 +17,7 @@ export default function ModifyProfileInput({
     <div className="flex flex-col gap-[10px] mb-5">
       <label htmlFor={props.id}>{label}</label>
       <input
-        className="w-[400px] h-10 rounded-[50px] border-gray-c8 dark:border-gray-c8/50 border text-sm focus:border-primary pl-[25px]"
+        className="w-[400px] h-10 rounded-[50px] border-gray-c8 dark:border-gray-c8/50 border text-sm focus:border-primary pl-[25px] pr-6"
```

Comment:
```suggestion
        className="w-[400px] h-10 rounded-[50px] border-gray-c8 dark:border-gray-c8/50 border text-sm focus:border-primary px-6"
```
그렇다면 pl과 pr를 합쳐 이것은 어떤가요ㅎㅎㅎ 1px 차이길래요!